.gitignore

```diff
@@ -4,3 +4,5 @@ build/
 # vim temporary files
 *.swp
 *.swo
+
+compile_commands.json
```

Comment:
The exported `compile_commands.json` file should be in the build directory.
Also, it seems that this is leftover from some point and that it should be removed.

Reply:
I have it in root because of YCM/clangd, I'll remove it.